app/src/main/java/grinnell/appdev/edu/lyles/Adapters/BeerAdapter.java

```diff
@@ -36,14 +42,20 @@ public View getView(int pos, View convertView, ViewGroup parent) {
             convertView = LayoutInflater.from(this.getContext()).inflate(R.layout.beer_list_item, parent, false);
```

Comment:
nit: Formatting. Please use proper level indentation.  See code guideline or my PR in Publications as seen here: https://github.com/GrinnellAppDev/Publications-Android/blob/develop/app/src/main/java/edu/grinnell/grinnell_publications_android/Activities/MainActivity.java#L105